portal/templates/profile_macros.html

```diff
@@ -1687,7 +1688,21 @@ <h4 id="communicationsLoc" class="profile-item-title index-item">{{ _("Communica
                 </div>
             </div>
         </div>
-    {%- endif %}
+    {%- elif person and person.has_role(ROLE.STAFF) and current_user and current_user.has_role(ROLE.STAFF_ADMIN)-%}
+        <div class="row">
+            <div class="col-md-12 col-xs-12">
+                <div class="profile-item-container">
+                    <h4 id="communicationsLoc" class="profile-item-title index-item">{{ _("Communications") }}</h4>
+                    <div class="flex">
+                        <div id="resetPasswordContainer" class="flex-item">
+                            <p class="text-muted communication-prompt">Send reset password email to staff</p>
```

Comment:
Aha! Found the untagged string you've been adding to make sure I actually read your code ;)